### DISCUSSION TITLE
Set heat flux on local region of an obstacle

### DISCUSSION THREAD

**Discussion by DOGzym** [firemodels/fds#11830] [General]
I don't know how to set heat flux on local region in my numerical simulation.

I've tried to divide the obstacle (PMMA) into two parts and set heat flux on one part by 'EXTERNAL FLUX', but it seems that fire spread happened only on the part which I set 'EXTERNAL FLUX', and another part is affected little. I attach my input file below. So, in my such kind simulation, how can I truly set 'EXTERNAL FLUX' on local region?

I'd appreciate a lot if you can help me. Thanks!  
[obstacle.txt](https://github.com/firemodels/fds/files/11476502/obstacle.txt)

**Comment by drjfloyd** [comment]
If fire is happening on the OBST with the SURF with EXTERNAL_FLUX, then you have set a local flux. At that point, fire spread to cells without EXTERNAL_FLUX will require that the fire provides enough heat to those cells for them to start pyrolyzing.  I suggest you add a BNDF of WALL TEMPERATURE or some WALL TEMPERATURE devices so that you can see how much the other OBST is heating up. My guess is that the fire on the OBST with EXTERNAL_FLUX is not large enough for flame spread to happen.
